src/components/Features/TimerButton.tsx

```diff
@@ -1,18 +1,28 @@
+import React from 'react';
+
 const getButtonColor = (isRunning: boolean): string =>
 	isRunning ? 'bg-red-500 hover:bg-red-700' : 'bg-blue-500 hover:bg-blue-700';
 
 const TimerButton: React.FunctionComponent<{
 	isRunning: boolean;
 	toggleTimer: () => void;
-}> = ({ isRunning, toggleTimer }) => {
+	resetTimer: () => void;
```

Comment:
Extract props to separate interface e.g. 
interface Props {
 isRunning: boolean..
...
}